bokeh/sphinxext/bokeh_gallery.py

```diff
@@ -41,7 +41,9 @@ def run(self):
         for detail in details_iter:
             src_path = abspath(join("..", detail['path']))
             dest_path = join(dest_dir, detail['name'] + ".py")
-            docname = join("docs", "gallery", detail['name'])
+
+            # sphinx pickeled env works only with backslash
```

Comment:
typo: should be "pickled", also the content of the comment is wrong? It should say "only works with forward slash" ?